### DISCUSSION TITLE
我的网站在github上部署的，请问我该如何使用waline？

### DISCUSSION THREAD

**Discussion by at2er** [walinejs/waline#844] [Q&A] [answered]
我的网站在github上部署的，请问我该如何使用waline？我目前只想在github上部署，因为Vercel自己生成的域名好像很长（我还没有使用过），但是我目前无法自己购买一个域名。

**Comment by lizheming** [comment, marked as answer]
Vercel 生成的域名是提供给网站的 API 接口，不是用来给用户访问的，所以长短没有关系。建议看下快速开始操作下比较简单哈 https://waline.js.org/guide/get-started.html

**Reply by at2er** [reply]
那么我是否需要自己购买一个域名？

**Reply by lizheming** [reply]
快速开始里没有让用户购买域名的~

**Reply by at2er** [reply]
我现在使用Vercel时遇到了一些问
![1646393937655](https://user-images.githubusercontent.com/98253084/156757143-920b0a52-2770-40dd-bd73-4e54252e3500.png)
题

**Reply by at2er** [reply]
请问我该怎么办？

**Reply by at2er** [reply]
或者可不可以在github上使用waline？

**Reply by lizheming** [reply]
请按照快速开始的步骤操作，如果有问题请提供有问题的地址和复现方式。Waline 是客户端服务端数据库三位一体的，需要服务端，可以使用 GitHub 做数据库，但不支持仅用 GitHub。

**Reply by Mister-Hope** [reply]
/doge 逐渐失去耐心hhh

看文档，如果你对文档的某一步操作不明白，截图描述你做了什么得到了什么不符的结果。

直白点说，正常使用不会产生这些问题。，所以我们目前不理解你这些问题怎么问出来的，也不知道该如何回答他。而且我有理由怀疑你在自己随便搞。如果你是个小白，严格follow文档的每一步，而不是自己想当然。祝好。
